### PR TITLE
Ux Multichain : Updated Select Action Modal

### PR DESCRIPTION
**Pull request by NidhiKJha**
This PR is to add the following update to Select Action Modal :

1. Add Hover to the entire cell i.e. to each selected action item
2. Updated Description for Stake

### Before
Screenshot 2023-09-06 at 6 15 36 PM

### After
Screenshot 2023-09-06 at 6 15 02 PM

## Manual Testing Steps
 - Hover area update

## Pre-merge author checklist

- [x] I've clearly explained:
  - [x] What problem this PR is solving
  - [x] How this problem was solved
  - [x] How reviewers can test my changes
- [ ] Sufficient automated test coverage has been added

## Pre-merge reviewer checklist

- [ ] Manual testing (e.g. pull and build branch, run in browser, test code being changed)
- [ ] PR is linked to the appropriate GitHub issue
- [ ] **IF** this PR fixes a bug in the release milestone, add this PR to the release milestone

If further QA is required (e.g. new feature, complex testing steps, large refactor), add the `Extension QA Board` label.

In this case, a QA Engineer approval will be be required.

### CODE REVIEW

**Review by DDDDDanica — DISMISSED** [review]

**Review by zone-live — DISMISSED** [review]

File: app/_locales/en/messages.json [hunk 1]
```diff
@@ -4268,7 +4268,7 @@
     "message": "Stake"
   },
   "stakeDescription": {
-    "message": "Hold up your crypto and earn potential profits"
+    "message": "Stake your Crypto to earn rewards"
```
Comment (georgewrmarshall) [inline app/_locales/en/messages.json:4271]: Is "Crypto" intentionally using title case here? cc @coreyjanssen

**Review by georgewrmarshall — COMMENTED** [review]
LGTM! Checked UI in storybook just one question regarding content that I'm unsure of

**Comment by coreyjanssen** [comment]
Good catch @georgewrmarshall! Should definitely be lowercase `crypto`.

File: app/_locales/en/messages.json (hunk 1 above)
Comment (NidhiKJha) [reply app/_locales/en/messages.json]: Nice catch! Updated here https://github.com/MetaMask/metamask-extension/pull/20755/commits/24410e4726d0bb19b5af26ce90a56d13ccd3fe07

**Review by vthomas13 — APPROVED** [review]

**Review by zone-live — APPROVED** [review]

**Pull request merged**